include/ffconf.h

```diff
@@ -97,9 +101,24 @@
 /   950 - Traditional Chinese (DBCS)
 */
 
-
+#if defined(CONFIG_FS_FATFS_LFN) && CONFIG_FS_FATFS_LFN == 1

```

Comment:
Kconfig has logic states mapped as follows: TRUE->exist and equal 1, FALSE->doesn't exist.
For existing Kconfig definiions checkig of its value as here rather is not practiced.
`#if defined(CONFIG_FS_FATFS_LFN)` is enough. Eventually IS_ENABLED() macro might be used.

Reply:
Removed `&& CONFIG_FS_FATFS_LFN == 1`

Reply:
Removed the second condition